ViperRX/ViperRX/CountryModule/View/CountryViewController.swift

```diff
@@ -0,0 +1,88 @@
+// CountryViewController.swift
+// Copyright © Alexandr T. All rights reserved.
+
+import UIKit
+
+/// Экран со списоком стран
+final class CountryViewController: UIViewController {
+    // MARK: - Public Properties
+
+    var countriesPresenter: CountryPresenterProtocol?
+
+    // MARK: - Initializers
+
+    init(countriesPresenter: CountryPresenterProtocol?) {
+        super.init(nibName: nil, bundle: nil)
+        self.countriesPresenter = countriesPresenter
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError(S.fatalErrorText)
+    }
+
+    // MARK: - Private Properties
+
+    private var tableView = UITableView()
+
+    // MARK: - Lifecycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .orange
+        setupUI()
+    }
+
+    // MARK: - Private Methods
+
+    private func setupUI() {
+        createTableView()
+    }
+
+    private func createTableView() {
+        tableView.frame = view.frame
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(CountryTableViewCell.self, forCellReuseIdentifier: S.countryCellIdentifier)
```

Comment:
```suggestion
        tableView.register(CountryTableViewCell.self, forCellReuseIdentifier: Constants.countryCellIdentifier)
```
переименуй в свифтгене, S непонятно как то